modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/IgniteSqlToRelConvertor.java

```diff
@@ -83,6 +87,28 @@ public class IgniteSqlToRelConvertor extends SqlToRelConverter {
         }
     }
 
+    @Override
+    protected RexNode convertExtendedExpression(
```

Comment:
This won't fix problem with BIGINT literals and this code should be removed. 
See comments below.